nengo/tests/test_connection.py

```diff
@@ -611,6 +612,23 @@ def test_set_eval_points(Simulator):
     Simulator(model)  # Builds fine
 
 
+@pytest.mark.parametrize('eval_points, radius', [
+    (UniformHypersphere().sample(500, 1), 0.5),
+    (UniformHypersphere().sample(500, 1), 1.),
+    (UniformHypersphere().sample(500, 1), 1.5)])
+def test_eval_points_scaling(Simulator, eval_points, radius, seed):
+    model = nengo.Network(seed=seed)
+    with model:
+        a = nengo.Ensemble(1, 1, radius=radius)
+        b = nengo.Ensemble(1, 1)
+        con = nengo.Connection(a, b, eval_points=eval_points)
+
+    sim = Simulator(model)
+    dists = np.linalg.norm(sim.data[con].eval_points, axis=1)
+    assert np.all(dists <= radius)
+    assert np.any(dists >= 0.9 * radius)
```

Comment:
My 2 cents: regression tests are useful, as long as they don't add undue length to the test suite. At the same time, we have a lot of overlapping tests at this point, and it would be good to reduce its length, or separate them into those tests that we should run all the time, and those that we only need to run before releases, etc. Issues for another PR though for sure.